programs/js_operation_serializer/main.cpp

```diff
@@ -122,7 +122,6 @@ template<> struct js_name<fc::uint160>         { static std::string name(){ retu
 template<> struct js_name<fc::sha224>          { static std::string name(){ return "bytes 28";   } };
 template<> struct js_name<fc::sha256>          { static std::string name(){ return "bytes 32";   } };
 template<> struct js_name<fc::unsigned_int>    { static std::string name(){ return "varuint32";  } };
-template<> struct js_name<fc::signed_int>      { static std::string name(){ return "varint32";   } };
```

Comment:
Actually JS code is using `varint32` everywhere but not `varuint32`. Will submit an issue there.

Reply:
Hm, should we rename that to varuint64?